docs/source/remote_data.rst

```diff
@@ -199,6 +199,32 @@ Available expiry dates can be accessed from the ``expiry_dates`` property.
 
 .. _remote_data.enigma:
 
+Quandl
```

Comment:
the quandl tag should be *before* the ``Quandle\n------\n``; and move the enigma one *before* ``Enigma``.